gear/gear/auth.py

```diff
@@ -54,6 +54,9 @@ def rest_authenticated_users_only(fun):
     async def wrapped(request, *args, **kwargs):
         userdata = await userdata_from_rest_request(request)
         if not userdata:
+            web_userdata = await userdata_from_web_request(request)
+            if web_userdata:
```

Comment:
Were these your changes or ones Dan has in a different pull request?

Reply:
the ones Dan has in a different pull request

Reply:
I'd say let Dan's PR go in with these changes and delete these lines. That way it's clear why they were added with the correct PR message.